plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -61,26 +62,41 @@ public void createControl(Composite parent) {
         AnalyticsEvents.APP_ENGINE_NEW_PROJECT_WIZARD_TYPE_NATIVE, parent.getShell());
 
     Composite container = (Composite) getControl();
+    GridLayoutFactory.fillDefaults().numColumns(1).extendedMargins(0, 0, 0, 8).applyTo(container);
```

Comment:
`fillDefaults()` has one column by default (cf. Javadoc), so `numColumns(1)` is unnecessary.

BTW, why do you have this code in this PR? I remember I saw it somewhere else.